reinfi/zf-dependency-injection
# Fix/psr 4 autoload violation

This should solve problems with future `composer dumpautoload -o` classmap generations.

How to test:
- checkout code (master)
- run `composer dumpautoload -o`

The following messages should appear:
```
Generating optimized autoload files
Deprecation Notice: Class Reinfi\DependencyInjection\Test\Unit\Service\AutoWiringServiceTest located in ./test/Unit/Service/AutoWiringServiceTest.php does not comply with psr-4 autoloading standard. It will not autoload anymore in C
omposer v1.11+. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class Reinfi\DependencyInjection\Test\Unit\Service\Extractor\YamlExtractorTest located in ./test/Unit/Service/Extractor/YamlExtractorTest.php does not comply with psr-4 autoloading standard. It will not autoload
anymore in Composer v1.11+. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class Reinfi\DependencyInjection\Test\Unit\Service\InjectionServiceTest located in ./test/Unit/Service/InjectionServiceTest.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Com
poser v1.11+. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:185
```

Validate fix:
- checkout PR code
- run `composer dumpautoload -o`

No notice should popup
